style.css

```diff
@@ -96,3 +121,182 @@ header {
   width: 1.5rem;
   height: 1.5rem;
 }
+
+#Contact-me {
+  background: #6070ff;
+  border-top-left-radius: 80px;
+  background-image: url("/image/headerbg.png");
+  background-size: cover;
+  background-position: 100% 10%;
+  background-repeat: no-repeat;
+  height: auto;
+  width: 100%;
+  padding: 16px;
+}
+
+.contact {
+  width: 327px;
+  height: 102px;
+  margin: auto;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact-title {
+  margin: 70px auto 16px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  padding: 1rem 0 0;
+  text-align: center;
+  color: #fff;
+}
+
+.container {
+  padding-top: 5rem;
+}
+
+.text-field {
+  height: 48px;
+  width: 327px;
+  margin: 0;
+  text-align: center;
+}
+
+#fname {
+  height: 48px;
+  width: 340px;
+  border-radius: 8px;
+}
+
+#lname {
+  margin-top: 1rem;
+  height: 48px;
+  width: 340px;
+  border-radius: 8px;
+}
+
+.subject {
+  margin: 0;
+}
+
+#subject {
+  border-radius: 8px;
+  height: 100px;
+  width: 340px;
+  margin-top: 2rem;
+  color: #b3bac5;
+}
+
+button {
+  width: 140px;
+  height: 48px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 10px;
+  border: 1px solid #dfe1e6;
+  text-align: center;
+  letter-spacing: 0.03em;
+  margin: auto;
+}
+
+.cards .card {
+  border-radius: 16px;
+  margin: 6rem auto 0;
+  height: auto;
+  width: 90%;
+  padding: 16px;
+  background: #fff;
+}
+
+.img {
+  flex: none;
+  padding: 0;
+  margin: auto;
+  height: 280px;
+  width: 100%;
+  border-radius: 8px;
+}
+
+.works {
+  display: flex;
+  align-items: center;
+}
+
+.works2 {
+  display: flex;
+  align-items: center;
+}
+
+li {
+  color: #7a869a;
+}
+
+.works2 li {
+  margin-right: 0.5rem !important;
+}
+
+.works li {
+  margin-right: 0.3rem !important;
+}
+
+#aboutme {
+  border-top-right-radius: 80px;
+  margin-top: 8rem;
+  height: auto;
+  width: 100%;
+  padding: 16px;
+  background: #fff;
+}
+
+.aboutme {
+  font-weight: 700;
+  width: 250px;
+  height: 101px;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+}
```

Comment:
- [ ] As the previous code reviewer said:

> Kindly ensure that the color, font family, font size, line height, margins (top, bottom, left, right) and font weight of your about section i is stick to the Figma design you chose as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). You can check out the Figma design specification for the exact properties used👍🏽
> I suggest importing this font family I show you below so that you can apply it to your all body 👍 😀 😀 😀

`@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");`

` font-family: "Poppins", sans-serif;`

- [ ] Please 🙏🏽 correct the font styles to look similar to the Figma template.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/192121804-02515669-f116-47b0-8168-4684e281a510.png) | ![image](https://user-images.githubusercontent.com/58642949/192121777-28e5cab0-20c1-484b-8466-c3fc8fd4f3b3.png) |

Reply:
After wasting much time, you came back with was not captured in the last review. It should not have been an obstacle for approval. Meanwhile, it is 12:23am (UTC+1), Sunday, here.

Thanks

Reply:
@franclobo